network/src/protocols/rpc/utils.rs

```diff
@@ -0,0 +1,38 @@
+use crate::{
```

Comment:
nit: do you want to have it as part of this diff? Taking into account that state sync won't actually use rpc calls

Reply:
Yeah, I'll split into a separate PR